nixos/modules/rename.nix

```diff
@@ -137,6 +137,9 @@ with lib;
     # tarsnap
     (mkRemovedOptionModule [ "services" "tarsnap" "cachedir" ] "Use services.tarsnap.archives.<name>.cachedir")
 
+    # unifi
+    (mkRemovedOptionModule [ "services" "unifi" "dataDir" ] "To use external storage, configure unifi to use an external Mongo instance and configure that accordingly.")
```

Comment:
I'm hesitating if we should eliminate this option.

You're right that we can now use `settings.properties` to configure the mongodb. As far as I can tell, however, the mongodb is only part of the data unifi stores in that directory. Parts of the NixOS community appreciates an ephemeral root filesystem (for example, by mounting `/` as `tmpfs`). Without the `dataDir` option we need a configuration to define all persistent paths. I'm not sure whether this is possible with `settings.properties` alone.